source/extensions/tracers/skywalking/skywalking_stats.h

```diff
@@ -17,6 +17,8 @@ struct SkyWalkingTracerStats {
   SKYWALKING_TRACER_STATS(GENERATE_COUNTER_STRUCT)
 };
 
+using SkyWalkingTracerStatsPtr = std::shared_ptr<SkyWalkingTracerStats>;
```

Comment:
Please use `SkyWalkingTracerStatsSharedPtr` here to ensure that the style is consistent with the community.

Reply:
ok,thanks.